src/main/java/com/stripe/net/StripeHeaders.java

```diff
@@ -0,0 +1,32 @@
+package com.stripe.net;
+
+import java.util.List;
+import java.util.Map;
+
+public class StripeHeaders {
+
+	Map<String, List<String>> headers;
+
+	public StripeHeaders(Map<String, List<String>> headers) {
+		this.headers = headers;
+	}
+
+	/**
+	 * Returns the first header value for a given key
+	 * @param name The name of the header key
+	 * @return the first value for the given key
+	 */
+	public String get(String name) {
+		List<String> valuesList = values(name);
+		String requestId = null;
```

Comment:
Sorry to be such a bother, but mind giving a generic name to this variable too?

Reply:
Thanks for the catch